lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -154,26 +154,34 @@ export class RefreshTokenClient extends BaseClient {
 
         parameterBuilder.addClientInfo();
 
-        const correlationId = request.correlationId || this.config.cryptoInterface.createNewGuid();
+        const correlationId = request.correlationId ? request.correlationId : this.config.cryptoInterface.createNewGuid();
```

Comment:
These are functionally equivalent, what was the warning/error?

Reply:
request.correlationId can be `undefined`